executor/agents/agno/mcp_manager.py

```
@@ -248,41 +300,48 @@ async def cleanup_tools(self) -> None:
-                if hasattr(tools, 'disconnect'):
+                if hasattr(tools, "disconnect"):
                     await tools.disconnect()
             except Exception as e:
                 logger.warning(f"Failed to disconnect MCP tools: {str(e)}")
                 # Add thinking step for MCP tools disconnection failure
                 self.add_thinking_step_by_key(
                     title_key="thinking.mcp_init_fail",
                     report_immediately=False,
-                    details={"error_message": f"Failed to disconnect MCP tools. \nerror message: {str(e)}. \ntools: {json.dumps(tools, ensure_ascii=False)}"}
+                    details={
+                        "error_message": f"Failed to disconnect MCP tools. \nerror message: {str(e)}. \ntools: {json.dumps(tools, ensure_ascii=False)}"
```

Ruff (0.14.8)

Use explicit conversion flag

Replace with conversion flag

(RUF010)